packages/lesswrong/server/resolvers/defaultResolvers.ts

```diff
@@ -61,18 +62,26 @@ const addDefaultResolvers = <N extends CollectionNameString>(
           enableCache?: boolean,
           enableTotal?: boolean,
           createIfMissing?: Partial<T>,
+          resolverArgs?: Record<string, unknown>,
         },
       },
       context: ResolverContext,
       info: GraphQLResolveInfo,
     ) => {
       // const startResolve = Date.now()
       const input = args?.input || {};
-      const { terms={}, enableCache = false, enableTotal = false } = input;
+      const { terms={}, enableCache = false, enableTotal = false, resolverArgs={} } = input;
       const logger = loggerConstructor(`views-${collectionName.toLowerCase()}-${terms.view?.toLowerCase() ?? 'default'}`)
       logger('multi resolver()')
       logger('multi terms', terms)
 
+      const termKeys = Object.keys(terms);
+      const overlappingKeys = Object.keys(resolverArgs).filter(termKey => termKeys.includes(termKey));
+
+      if (overlappingKeys.length) {
+        captureException(`Got a ${collectionName} multi request with overlapping term and resolverArg keys: ${overlappingKeys.join(', ')}`);
```

Comment:
Making a choice to bail out here.

Right now, resolver args just don't get passed through at all.  This PR fixes that, and my best guess is that we haven't introduced any cases where there are resolverArgs (extraVariableValues) that have the same field names as terms.  (I tried doing some type magic to catch any overlap at the callsite, but there are too many places where we're passing through terms/etc through multiple layers of indirection and it was too much headache.  It did let me check most uses, though, and we very rarely pass in `extraVariableValues` to `useMulti` in the first place.)  It seems better to catch any accidental overlap we introduce by failing in a very obvious way, rather than silently defaulting to the value in `terms` for any overlapping fields and hoping we catch it in the logs.